src/Ballot.php

```diff
@@ -6,4 +6,11 @@ class Ballot
     /**
      * @todo #4 Implement ballot logic
      */
+	private $listOfListOfCandidates;
+
+  	public function __construct(CandidateListList /*...*/$listsOfCandidates) {
```

Comment:
I just realized I have php 5.5, so I can't leverage php 5.6+'s varargs syntax yet. I will upgrade soon and uncomment the ellipses.